components/dashboard/src/start/StartPage.tsx

```diff
@@ -139,3 +142,16 @@ function StartError(props: { error: StartWorkspaceError }) {
     }
     return <p className="text-base text-gitpod-red w-96">{error.message}</p>;
 }
+
+function OrgDefaultWorkspaceWarningView(props: { workspaceId: string }) {
+    const { data: imageInfo } = useDefaultWorkspaceImageQuery(props.workspaceId);
+    if (imageInfo?.source !== "organization") {
+        return null;
+    }
+    return (
+        <Alert className="w-96 mt-4" type="warning">
+            A custom <span className="font-medium">default workspace image</span> is set for this organization. Contact
+            an organization owner or specify a different image in <code>.gitpod.yml</code>.
+        </Alert>
```

Comment:
We may need to update this warning message since we can't identify if current workspace were started with **org default workspace**. See _How to Test_ section _Not_ part. 

cc @gtsiolis @filiptronicek If you have good idea about it